src/forms/index.tsx

```diff
@@ -252,19 +324,24 @@ export interface IFileInputProps {
   size?: "small" | "medium" | "large";
   align?: keyof typeof ALIGNMENTS;
   filename?: string;
+  filenames?: string[];
```

Comment:
Is there a reason for supporting both `filename` and `filenames`? It would clear up the interface to only support the latter, as you can get the behavior of the former with passing a single value array to `filenames`.

Reply:
I'd prefer exclusively using `filenames` as well. I kept `filename` because that API is currently in use and doesn't have a deprecation warning. I'll add a warning similar to the other items.